### PR TITLE
updated Pt_br

### PR DESCRIPTION
**Pull request by FITFC**

### CODE REVIEW

**Comment by Snownee** [comment]
There are some new entries that you did not translate.

Here is a merged version: https://gist.github.com/Snownee/aa216156fa4e47fde0049c952a7de54d

File: src/main/resources/assets/jade/lang/pt_br.json [hunk 1]
```diff
@@ -1,208 +1,201 @@
...
+  "config.jade.plugin_minecraft.harvest_tool.new_line": "Comece em nova linha",
+  "config.jade.plugin_minecraft.harvest_tool.effective_tool": "Ferramenta eficaz",
+  "jade.harvest_tool.unbreakable": "Não posso quebrar",
+  "config.jade.plugin_minecraft.enchantment_power": "Poder de encantamento",
+  "config.jade.plugin_minecraft.total_enchantment_power": "Poder de encantamento total",
+  "jade.ench_power": "Poder do Encantamento: %s",
+  "config.jade.plugin_minecraft.player_head": "Cabeça de jogador",
+  "config.jade.plugin_minecraft.profession": "Profissão de aldeão",
+  "config.jade.plugin_minecraft.item_tooltip": "Dica de ferramenta",
+  "config.jade.plugin_minecraft.block_inventory": "Item Storage",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/pt_br.json]: this

File: src/main/resources/assets/jade/lang/pt_br.json [hunk 2]
```diff
@@ -1,208 +1,201 @@
...
+  "config.jade.overlay_scale": "Escala de exibição",
+  "config.jade.overlay_anchor_x": "Âncora horizontal",
+  "config.jade.overlay_anchor_y": "Âncora vertical",
+  "config.jade.overlay_square": "Borda quadrada",
+  "config.jade.flip_main_hand": "Vire a posição com a mão principal",
+  "config.jade.flip_main_hand_desc": "Virar a posição da dica de ferramenta se a mão principal for deixada",
+  "config.jade.overlay_color": "Cor",
+  "config.jade.overlay_alpha": "Alfa",
+  "config.jade.overlay_theme": "Tema",
+  "config.jade.animation": "Animation",
```
Comment (Snownee) [inline src/main/resources/assets/jade/lang/pt_br.json]: this

**Review by Snownee — CHANGES_REQUESTED** [review]

**Comment by FITFC** [comment]
Thanks for the help, I had really looked everything up and hadn't seen this

**Comment by Snownee** [comment]
Thank you!

**Pull request merged**